Sources/SmileID/Classes/Networking/Models/Authentication.swift

```diff
@@ -22,25 +22,47 @@ public struct AuthenticationRequest: Codable {
         case partnerId = "partner_id"
         case authToken = "auth_token"
     }
-}
 
-public extension AuthenticationRequest {
-    init(jobType: JobType, enrollment: Bool, updateEnrolledImage: Bool, jobId: String, userId: String) {
-        self.init(jobType: jobType,
-                  enrollment: enrollment,
-                  updateEnrolledImage: updateEnrolledImage,
-                  jobId: jobId,
-                  userId: userId,
-                  signature: true)
+    public init(jobType: JobType,
+                enrollment: Bool,
+                updateEnrolledImage: Bool?,
+                jobId: String?,
+                userId: String?,
+                signature: Bool,
+                production: Bool,
+                partnerId: String,
+                authToken: String) {
```

Comment:
Is it possible to provide defaults here, so that we don't need a 2nd initializer?

i.e. something like:
```suggestion
                production: Bool = !SmileID.useSandbox,
                partnerId: String = SmileID.config.partnerId,
                authToken: String = SmileID.config.authToken) {
```

Reply:
Done